### DISCUSSION TITLE
Chia is dying, within 30 days netspace dropped by 3.34 EB - Simple solution

### DISCUSSION THREAD

**Discussion by majekqwert** [Chia-Network/chia-blockchain#17858] [General]
Chia is a great project that is constantly evolving. Unfortunately, its financial support from outside sources is minimal. YouTube videos are viewed by fewer than 1,000 people, and 90% of the comments are from fake accounts. If things continue this way, the project will collapse in about six months. The whole problem with Chia lies in its massive pre-farming. Currently, the Chia creators own about 70% of all Chia, which effectively deters investors because no one knows if Bram will come up with the idea of needing money and sell them all in a day or two. I understand that project development costs money, and its creators also want to make a profit, but this needs to be resolved differently and more transparently. At this point, it seems to me that the current state of pre-farming (what's left of it) should be public and easily accessible. In addition, these funds should be locked using smart transactions and, for example, paid out in small amounts each month. This way, the project will have support, creators will make profits, and investors will have confidence that the creators won't crash the project within a few hours by selling their Chia. I leave this for discussion, what do you think about it?
![Zrzut ekranu 2024-04-11 183130](https://github.com/Chia-Network/chia-blockchain/assets/21694511/47b86839-d4d7-44dd-88d0-85d6fb9cdcc5)

**Comment by BrandtH22** [comment]
The prefarm is locked in a custody solution that has onchain controls and is publicly auditable. Community sites like chia tracker have added the ability to track the prefarm in a user friendly manner and for those who want to deep dive and audit themselves there is this guide on how to audit the prefarm:
https://docs.chia.net/custody/prefarm-audit-tutorial/

Lastly, if you expand your timescale you will find that the drop in netspace goes against the greater trend of increasing netspace:
https://dashboard.chia.net/d/CL1X4UWnk/blockchain-state?orgId=1&from=now-6M&to=now&viewPanel=6

**Comment by BrandtH22** [comment]
Closed due to no response from OP
